src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.mapper.CinemaHallMapper;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private CinemaHallMapper cinemaHallMapper;
+    private CinemaHallService cinemaHallService;
+
+    @Autowired
+    public CinemaHallController(CinemaHallMapper cinemaHallMapper,
```

Comment:
```suggestion
    public CinemaHallController(Mapper<CinemaHall, CinemaHallRequestDto> cinemaHallRequestMapper,
```
and response mapper separately